meta-rcar-gen3-xen/recipes-core/images/core-image-minimal.bbappend

```diff
@@ -1,10 +1,20 @@
 #Add Xen to build
 IMAGE_INSTALL_append = " \
     xen-base \
-    xen-flask \
```

Comment:
any reason we remove flask? wouldn't it be needed for someone in this reference build?